test/pyrostest/test_system.py

```diff
@@ -0,0 +1,63 @@
+import pytest
+
+import pyrostest
+
```

Comment:
Its important tho.

Also IDK why but this is flaky in a really awful way. The spin forever node is being badly behaved.